### PR TITLE
Vendor for beatless

### PR DESCRIPTION
**Pull request by ph**
Add goformation which is the cloudformation documents.
Remove aws/lambda
Remove aws/cloudwatchlogs
add aws/cloudformation
add go-jmespath 
add yaml dependenceis for goformation

### CODE REVIEW

File: vendor/vendor.json [hunk 1]
```diff
@@ -1623,6 +1655,22 @@
...
+			"versionExact": "v2"
+		},
+		{
+			"checksumSHA1": "qoSnctxGqjdm6K9unlf/Mgo+mbA=",
+			"path": "github.com/sanathkr/yaml",
+			"revision": "0ca9ea5df5451ffdf184b4428c902747c2c11cd7",
+			"revisionTime": "2017-03-27T23:54:44Z",
+			"version": "v1.0.0",
+			"versionExact": "v1.0.0"
+		},
```
Comment (urso) [inline vendor/vendor.json]: How come this PR introduces 2 yaml libs?

Comment (ph) [reply vendor/vendor.json]: These are two dependencies of goformation they overrides some behavior on the tags.

https://github.com/awslabs/goformation/blob/master/go.mod#L13-L14

Comment (ph) [reply vendor/vendor.json]: @urso I wish we didn't have to bring the yaml dependencies, I only use the JSON format :(

**Comment by ph** [comment]
CI errors is not related to this PR, but I will do a followup on them.

**Pull request merged**